packages/wallets/wallet-turnkey/src/strategy/strategy.ts

```
@@ -137,19 +148,67 @@ export class TurnkeyWalletStrategy
+              http: [url],
+            },
+          },
+        },
+        transport: http(url),
+      })
+
+      const preparedTransaction = await accountClient.prepareTransactionRequest(
+        transaction as PrepareTransactionRequestParameters,
+      )
+
+      delete preparedTransaction.account
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)